themes/src/main/resources/theme/keycloak.v2/login/resources/css/styles.css

```diff
@@ -79,6 +79,11 @@ div.kc-logo-text span {
     overflow-wrap: break-word
 }
 
+#kc-verify-email-form {
+    margin-top: 24px;
+    margin-bottom: 24px;
+}
+
```

Comment:
we want to minimize the amount of custom css, is this really needed it should look fine with default patternfly

Reply:
right, because this is in v1 it's fine to leave it like it is